Resources/config/authorize.xml

```diff
@@ -20,7 +20,21 @@
 
         <service id="fos_oauth_server.authorize.form.handler.default" class="FOS\OAuthServerBundle\Form\Handler\AuthorizeFormHandler">
             <argument type="service" id="fos_oauth_server.authorize.form" />
-            <argument type="service" id="request_stack" on-invalid="ignore" />
+            <argument type="service" id="request_stack" />
```

Comment:
Since you're already breaking BC by removing `setContainer`, you should also change the signature of `AuthorizeFormHandler::__construct`. The other option is keeping this optional and triggering a deprecation notice that the argument will be required soon.